cypress/test/newsroom/NewsroomSection.cy.js

```diff
@@ -0,0 +1,25 @@
+import { mount } from 'cypress/react'
+import NewsroomSection from '../../../components/newsroom/NewsroomSection';
+const newsroom = ["Latest news and blogs", "Welcome to our Newsroom section. Here, you'll get latest information about our blogs, articles, announcements and Youtube live-streams. Let's get upto date with the recent activities in the organization."]
```

Comment:
Only check for the heading, the content inside heading can be modified later on.

Reply:
done

Reply:
This is not updated in implementation yet.

Reply:
done